points added for theme practices (#6710)
magento/devdocs: src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

   <a href="#"><?= __('Click to download'); ?></a>
   ```

1. Make use of the mobile first approach when inheriting blank or luma themes which has already been followed in these themes.

> 1. Mangento have a set of [coding stadrads]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies, Do refer and follow them when needed.

Reviewer: ```suggestion
1. Magento has a set of [coding standards]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies, Do refer and follow them when needed.
```

> 1. Do not repeat a work, instead create a class or method and call them when needed.

Reviewer: I have doubts regarding this point. If we are talking about PHP classes/methods, they are not supposed to be a part of a theme. They may, but not in the classic Magento Theme definition. So, I would suggest to remove it.

Author: Yes, that is correct. will change that. thanks.

> 1. While styling any custom module, Do add the styling for that specific module instead of adding it in the design theme so the style will not get loaded unless the module is being used. For example, like in app/code/Company/Module/view/frontend/web/css/source/_module.less.

Reviewer: ```suggestion
1. While styling any custom module, do add the styling for that specific module instead of adding it in the design theme so the style will not get loaded unless the module is being used. For example, like in app/code/Company/Module/view/frontend/web/css/source/_module.less.
```

Author: Thanks for the suggestion @rogyar. I guess I have fixed the error. Please review.
